website/static/js/fangorn.js

```diff
@@ -209,30 +213,30 @@ function resolveIconView(item) {
         var template = m('span', { 'class' : iconType});
         return template;
     }
-    if (item.data.nodeType === 'smartFolder') {
-        return returnView('smartCollection');
-    }
-    if (item.data.nodeType === 'folder') {
+    if (item.data.isDashboard) {
         return returnView('collection');
     }
-    if (item.data.nodeType === 'pointer' && item.parent().data.nodeType !== 'folder') {
+    if (item.data.nodeType === 'smartFolder' || item.data.isSmartFolder) {
```

Comment:
This seems incorrect; nodeType can neve be `smartFolder`; see https://github.com/CenterForOpenScience/osf.io/blob/33ebadfa50e1a92922113a32e0658bba05db7541/website/util/rubeus.py#L368